monolith/classes/test/test_die.py

```diff
@@ -26,7 +26,3 @@ def test_multi_face_die(self):
     def test_die_to_str(self):
         die = Die(["1", "2", "3"], "test_theme")
         self.assertEqual(str(die), "['1', '2', '3']")
-
- 
-if __name__ == '__main__':
-    unittest.main()

```

Comment:
don't we need these?

Reply:
The tests correctly run either way, but the coverage says these lines aren't tested, so I'm guessing they are not really needed